Source/SwiftLintFramework/Configuration.swift

```diff
@@ -112,7 +113,7 @@ public struct Configuration {
                 let yamlContents = try NSString(contentsOfFile: fullPath,
                     encoding: NSUTF8StringEncoding) as String
                 if let _ = Configuration(yaml: yamlContents) {
-                    print("Loading configuration from '\(path)'")
+                    fputs("Loading configuration from '\(path)'\n", stderr)
```

Comment:
I didn't change this one originally since it didn't really seem like an "error" thoughts?